File: bulk/csharp/WeakRng01.cs
// CWE-330: System.Random for security-sensitive token
using System;
public class WeakRng01 {
    public string GenerateToken() {
        var rng = new Random();
        return rng.Next().ToString("x8") + rng.Next().ToString("x8"); // not cryptographic
    }
}

File: bulk/java/WeakRng01.java
// CWE-330: java.util.Random used for security-sensitive token
import java.util.Random;
public class WeakRng01 {
    public String generateResetToken() {
        Random rng = new Random(); // not cryptographically secure
        return Long.toHexString(rng.nextLong());
    }
}
